dotcom-rendering/src/components/Standfirst.tsx

```diff
@@ -202,12 +242,14 @@ const hoverStyles = css`
 `;
 
 export const Standfirst = ({ format, standfirst }: Props) => {
+	if (standfirst === '') return null;
```

Comment:
@mxdvl I added this since your review, and I wanted to check if you're ok with it?

Otherwise, we see padding for the standfirst container even when there's no standfirst content. This makes the gap between headline & image/article body look really weird!

Reply:
This is very sensible. We might want to prevent whitespace-only text as well:

```suggestion
	if (standfirst.trim() === '') return null;
```